<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>785</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PS Ca√ßador de Recursos - CoC</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../midia/logo/ps.png</normaloff>../midia/logo/ps.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="gridWidget" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>701</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="3">
      <widget class="QLineEdit" name="lineEdit_gold">
       <property name="font">
        <font>
         <family>Arial Black</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
border: 2px solid rgb(45, 45, 45);
border-radius: 10px;
padding: 15px;
background-color: rgb(104, 4, 19);
}

QLineEdit:hover {
border: 2px solid rgb(45, 45, 45);
background-color: rgb(114, 14, 29);
}

QLineEdit:focus {
border: 2px solid rgb(252, 230, 5);
background-color: rgb(114, 14, 29);
color: rgb(255, 255, 255)
}
</string>
       </property>
       <property name="placeholderText">
        <string>qtd de ouro a procurar</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="img_ouro">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../midia/images/gold.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="img_dark">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../midia/images/dark_elixir.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="lineEdit_dark">
       <property name="font">
        <font>
         <family>Arial Black</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
border: 2px solid rgb(45, 45, 45);
border-radius: 10px;
padding: 15px;
background-color: rgb(104, 4, 19);
}

QLineEdit:hover {
border: 2px solid rgb(45, 45, 45);
background-color: rgb(114, 14, 29);
}

QLineEdit:focus {
border: 2px solid rgb(252, 230, 5);
background-color: rgb(114, 14, 29);
color: rgb(255, 255, 255)
}
</string>
       </property>
       <property name="placeholderText">
        <string>qtd de elixir thiago a procurar</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="lineEdit_elixir">
       <property name="font">
        <font>
         <family>Arial Black</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
border: 2px solid rgb(45, 45, 45);
border-radius: 10px;
padding: 15px;
background-color: rgb(104, 4, 19);
}

QLineEdit:hover {
border: 2px solid rgb(45, 45, 45);
background-color: rgb(114, 14, 29);
}

QLineEdit:focus {
border: 2px solid rgb(252, 230, 5);
background-color: rgb(114, 14, 29);
color: rgb(255, 255, 255)
}
</string>
       </property>
       <property name="placeholderText">
        <string>qtd de elixir a procurar</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="img_elixir">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../midia/images/elixir.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="reset_dark">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgb(207, 187, 4);
border: 2px solid rgb(67, 2, 12);
border-radius: 5px;
color: rgb(255, 255, 255);
}


QPushButton:hover{
background-color: rgb(114, 4, 19);
border: 2px solid rgb(77, 12, 22);
}

QPushButton:pressed{
background-color: rgb(252, 230, 5);
border: 2px solid rgb(57, 0, 2);
color: rgb(0, 0, 0)
}

QPushButton:disabled {
background-color: #cccccc; /* Cinza */
 border: 2px solid #999999;
color: #666666; /* Texto cinza escuro */
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../midia/images/reset_png.png</normaloff>../midia/images/reset_png.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="reset_elixir">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgb(207, 187, 4);
border: 2px solid rgb(67, 2, 12);
border-radius: 5px;
color: rgb(255, 255, 255);
}


QPushButton:hover{
background-color: rgb(114, 4, 19);
border: 2px solid rgb(77, 12, 22);
}

QPushButton:pressed{
background-color: rgb(252, 230, 5);
border: 2px solid rgb(57, 0, 2);
color: rgb(0, 0, 0)
}

QPushButton:disabled {
background-color: #cccccc; /* Cinza */
 border: 2px solid #999999;
color: #666666; /* Texto cinza escuro */
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../midia/images/reset_png.png</normaloff>../midia/images/reset_png.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="reset_gold">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgb(207, 187, 4);
border: 2px solid rgb(67, 2, 12);
border-radius: 5px;
color: rgb(255, 255, 255);
}


QPushButton:hover{
background-color: rgb(114, 4, 19);
border: 2px solid rgb(77, 12, 22);
}

QPushButton:pressed{
background-color: rgb(252, 230, 5);
border: 2px solid rgb(57, 0, 2);
color: rgb(0, 0, 0)
}

QPushButton:disabled {
background-color: #cccccc; /* Cinza */
 border: 2px solid #999999;
color: #666666; /* Texto cinza escuro */
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../midia/images/reset_png.png</normaloff>../midia/images/reset_png.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Arial Black</family>
         <pointsize>22</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>OURO</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Arial Black</family>
         <pointsize>22</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ELIXIR</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Arial Black</family>
         <pointsize>22</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ELIXIR THIAGO</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="moldura">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>721</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(252, 230, 5);
border: 4px solid rgb(0, 0, 0);
border-radius: 30px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fundo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>785</width>
      <height>600</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>785</width>
      <height>600</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>785</width>
      <height>600</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(132, 6, 23);
border: 5px solid rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>545</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../midia/logo/ps.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>449</y>
      <width>771</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="procurar">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Arial Black</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgb(104, 4, 19);
border: 2px solid rgb(67, 2, 12);
border-radius: 10px;
color: rgb(255, 255, 255);
}


QPushButton:hover{
background-color: rgb(114, 4, 19);
border: 2px solid rgb(77, 12, 22);
}

QPushButton:pressed{
background-color: rgb(252, 230, 5);
border: 2px solid rgb(57, 0, 2);
color: rgb(0, 0, 0)
}

QPushButton:disabled {
background-color: #cccccc; /* Cinza */
 border: 2px solid #999999;
color: #666666; /* Texto cinza escuro */
}
</string>
       </property>
       <property name="text">
        <string>Procurar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="parar">
       <property name="font">
        <font>
         <family>Arial Black</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgb(104, 4, 19);
border: 2px solid rgb(67, 2, 12);
border-radius: 10px;
color: rgb(255, 255, 255);
}


QPushButton:hover{
background-color: rgb(114, 4, 19);
border: 2px solid rgb(77, 12, 22);
}

QPushButton:pressed{
background-color: rgb(252, 230, 5);
border: 2px solid rgb(57, 0, 2);
color: rgb(0, 0, 0)
}

QPushButton:disabled {
background-color: #cccccc; /* Cinza */
 border: 2px solid #999999;
color: #666666; /* Texto cinza escuro */
}
</string>
       </property>
       <property name="text">
        <string>Parar</string>
       </property>
       <property name="shortcut">
        <string>F10</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>12</y>
      <width>771</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>9</pointsize>
      <weight>10</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 9pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;PS&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Ca√ßador de Recursos - CoC&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <zorder>fundo</zorder>
   <zorder>moldura</zorder>
   <zorder>gridWidget</zorder>
   <zorder>logo</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>label_4</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>785</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
